widget-reference/chart.md

```diff
@@ -61,6 +63,17 @@ description: >-
       <td style="text-align:left">Controls widget&apos;s visibility on the page. When turned off, the widget
         will not be visible when the app is published</td>
     </tr>
+    <tr>
+      <td style="text-align:left"><b>Add chart</b>
+      </td>
+      <td style="text-align:left">Multiple charts can be added for one widget. By clicking this button, you can enter the data and it will be displayed as a separate chart.</td>
```

Comment:
@ukitta555 can you modify this line to say multiple series of data can be displayed in a single chart widget using the Add series button